coordinator/core/src/test/kotlin/net/consensys/zkevm/ethereum/coordination/conflation/GlobalBlobAwareConflationCalculatorTest.kt

```diff
@@ -86,7 +87,8 @@ class GlobalBlobAwareConflationCalculatorTest {
     calculator = GlobalBlobAwareConflationCalculator(
       conflationCalculator = globalCalculator,
       blobCalculator = calculatorByDataCompressed,
-      batchesLimit = defaultBatchesLimit
+      batchesLimit = defaultBatchesLimit,
+      metricsFacade = mock<MetricsFacade>(defaultAnswer = Mockito.RETURNS_DEEP_STUBS)
```

Comment:
Same goes here.


Also, we should add tests to check that the metrics are correct.

Reply:
Agree to add tests for the metrics and I'd use mocks here for `metricsFacade` and histogram instances to check the metric values passed to the histogram `record` function